From 5559017bf01aa6127bf2d6b838da324062e9e080 Mon Sep 17 00:00:00 2001
From: Pavan Kumar S <pavan.kumar.s@intel.com>
Date: Wed, 15 May 2013 20:21:43 +0530
Subject: [PORT FROM MAIN] ANDROID-COS: Add a COS-MOS transition threshold with indication

BZ: 49173

This patch adds an additional check in COS that we allow boot to MOS only
with atleast a minimum capacity. As the minimum thresholds for booting to
COS & MOS are different, we should not allow the transition from COS to MOS
before a certain minimum battery capacity.

Orig-Change-Id: Ia4a534ddb0151fafe079e663e5cea92d9a7ce6c9

Change-Id: Ic914232fed8ef5e090595acc4358f69c47c33ebb
Signed-off-by: Pavan Kumar S <pavan.kumar.s@intel.com>
---
 board/redhookbay/init.redhookbay.rc   |    1 +
 board/victoriabay/init.victoriabay.rc |    1 +
 2 files changed, 2 insertions(+), 0 deletions(-)

diff --git a/board/redhookbay/init.redhookbay.rc b/board/redhookbay/init.redhookbay.rc
index f66956f..a0d7d7e 100755
--- a/board/redhookbay/init.redhookbay.rc
+++ b/board/redhookbay/init.redhookbay.rc
@@ -197,6 +197,7 @@ on property:sys.boot_completed=1
 on charger
     mount_all /fstab.charger.${ro.hardware}
 
+    setprop ro.boot.min.cap 3
     start watchdogd
     insmod /lib/modules/rmi4.ko boot_mode=1
 
diff --git a/board/victoriabay/init.victoriabay.rc b/board/victoriabay/init.victoriabay.rc
index f4f099e..911e9ea 100755
--- a/board/victoriabay/init.victoriabay.rc
+++ b/board/victoriabay/init.victoriabay.rc
@@ -162,6 +162,7 @@ on property:sys.boot_completed=1
 on charger
     mount_all /fstab.charger.${ro.hardware}
 
+    setprop ro.boot.min.cap 3
     start watchdogd
     insmod /lib/modules/rmi4.ko boot_mode=1
 
-- 
1.7.4.1

